library/CHART-TEMPLATE/templates/sidecar-deployment.yaml

```diff
@@ -16,7 +16,6 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       labels:
-        {{- include "common.labels.selectorLabels" . | nindent 8 }}
         {{- include "common.labels.podLabels" . | nindent 8 }}
```

Comment:
Personal opinion, what do we win with having the selectorLabels inside the podLabels? For me it ofuscates the fact that the podLabels have the selectorLabels and will match the rule.

Reply:
I agree that they are obfuscated and that even the user can footgun itself overriding labels that are used in the `selectorLabels` using `.global.podLabels` and `.podLabels`.

On the other hand, it is a matter of consistency, in how Yaml templating and Yaml parser work, both of the dictionaries should have to be merged or duplicated key-values could create issues depending on how the helpers are arranged.